### ISSUE TITLE
New "Deceased" field format cannot be parsed

### ISSUE THREAD

**Issue by rgreinho** [scrapd/scrapd#80]
Issue Type
----------

- Bug report

Current Behavior
----------------

If the parts or the "Deceased" field are separated by pipes (`|`), the parser fails extract the information.

Example of failing reports:
* https://austintexas.gov/news/traffic-fatality-15-4
* https://austintexas.gov/news/traffic-fatality-16-4
* https://austintexas.gov/news/traffic-fatality-17-4

Expected Behavior
-----------------

The parts should be extracted correctly.

**Comment by rgreinho** [comment]
The Regex for extracting the deceased field needs to be updated.

This may help: https://regex101.com/r/1sEvb1/2

**Comment by rgreinho** [comment]
Here is the regex which seems to work: `>Deceased:\s*</strong>\s*>?([^<]*\d)\s*.*\)?<`

**Comment by rgreinho** [comment]
Ok, here is a patch that helps with that part:

```patch
diff --git a/scrapd/core/apd.py b/scrapd/core/apd.py
index 3545668..5ea55a2 100644
--- a/scrapd/core/apd.py
+++ b/scrapd/core/apd.py
@@ -326,7 +326,7 @@ def parse_page_content(detail_page, notes_parsed=False):
         (Fields.CASE, re.compile(r'Case:.*\s([0-9\-]+)<')),
         (Fields.CRASHES, re.compile(r'Traffic Fatality #(\d{1,3})')),
         (Fields.DATE, re.compile(r'>Date:.*\s{2,}([^<]*)</')),
-        (Fields.DECEASED, re.compile(r'>Deceased:.*\s{2,}([^<]*\d)\)?<')),
+        (Fields.DECEASED, re.compile(r'>Deceased:\s*(?:</span>)?(?:</strong>)?\s*>?([^<]*\d)\s*.*\)?<')),
         (Fields.LOCATION, re.compile(r'>Location:.*>\s{2,}([^<]+)')),
         (Fields.TIME, re.compile(r'>Time:.*>\s{2,}([^<]+)')),
     ]
diff --git a/tests/core/test_apd.py b/tests/core/test_apd.py
index 5c38526..b35e3ab 100644
--- a/tests/core/test_apd.py
+++ b/tests/core/test_apd.py
@@ -70,6 +70,7 @@ parse_page_content_scenarios = {
     'traffic-fatality-2-3': {
         **parse_twitter_fields_scenarios['traffic-fatality-2-3'],
         Fields.AGE: 58,
+        Fields.CASE: '19-0161105',
         Fields.CRASHES: '2',
         Fields.DOB: '02/15/1960',
         Fields.DATE: '01/16/2019',

```

**Comment by rgreinho** [comment]
Closed by #90.
